Mathlib/GroupTheory/GroupAction/Blocks.lean

```diff
@@ -381,6 +381,93 @@ theorem IsBlockSystem.of_normal {N : Subgroup G} [N.Normal] :
 
 end Normal
 
+section Stabilizer
+
+/- For transitive actions, construction of the lattice equivalence
+  `block_stabilizerOrderIso` between
+  - blocks of `MulAction G X` containing a point `a ∈ X`,
+  and
+  - subgroups of G containing `stabilizer G a`.
+  (Wielandt, th. 7.5) -/
+
+/-- The orbit of a under a subgroup containing the stabilizer of a
+ is a block -/
```

Comment:
```suggestion
/-- The orbit of `a` under a subgroup containing the stabilizer of `a` is a block -/
```

Reply:
(As you see, I have a not so wide screen and I'm a very 80 column guy… I'll follow your recommendations anyway.)

Reply:
Ah, that makes sense :)